lewis/examples/simple_device/__init__.py

```diff
@@ -39,8 +39,7 @@ class VerySimpleInterface(StreamAdapter):
     After that, typing either of the commands and pressing enter sends them to the server.
     """
     commands = {
-        Cmd('get_param', '^P$'),
-        Cmd('set_param', '^P=(.+)$'),
+        Var('param', read_regex='^P$', write_regex='^P=(.+)$')
```

Comment:
Maybe keep the `Cmd` version and make the regex here match `V` instead (while still exposing `param`) just to illustrate both are possible.